hack/update-generated-swagger-spec.sh

```diff
@@ -32,40 +31,29 @@ trap "cleanup" EXIT
 
 set -e
 
-ADDR=127.0.0.1:8443
-HOST=https://${ADDR}
-SWAGGER_SPEC_REL_DIR=${1:-""}
-SWAGGER_SPEC_OUT_DIR="${OS_ROOT}/${SWAGGER_SPEC_REL_DIR}/api/swagger-spec"
-mkdir -p "${SWAGGER_SPEC_OUT_DIR}" || echo $? > /dev/null
-SWAGGER_API_PATH="${HOST}/swaggerapi/"
-
-# Prevent user environment from colliding with the test setup
-unset KUBECONFIG
 
-openshift=$(cd "${OS_ROOT}"; echo "$(pwd)/_output/local/bin/$(os::util::host_platform)/openshift")
+TMPDIR="${TMPDIR:-"/tmp"}"
+BASETMPDIR="${TMPDIR}/openshift-swagger"
+export ALL_IP_ADDRESSES=127.0.0.1
+export SERVER_HOSTNAME_LIST=127.0.0.1
+export API_HOST=127.0.0.1
+export API_PORT=38443
+export ETCD_PORT=34001
+export ETCD_PEER_PORT=37001
+export SUDO=''
+setup_env_vars
+export SUDO=''
```

Comment:
This looks redundant.  Are we anticipating that `setup_env_vars` could both read and assign `SUDO`?